decidim-admin/app/queries/decidim/admin/process_admins_roles_for_process.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A class used to find the roles of the users that can manage a given
+    # participatory process in a process admin role (that is, processs that are
+    # not organization admins).
+    class ProcessAdminsRolesForProcess
```

Comment:
Same as in `ManageableParticipatoryProcessesForUser` - I think this should be a `Rectify::Query`.